cirq-core/cirq/vis/heatmap.py

```diff
@@ -50,6 +53,12 @@ def __iter__(self):
         return iter(astuple(self))
 
 
+def _as_set(qubits: Iterable['cirq.Qid']) -> FrozenSet['cirq.Qid']:
+    if isinstance(qubits, np.ndarray):
+        return frozenset(qubits.reshape((-1,)).tolist())
+    return frozenset(qubits)
```

Comment:
This seems redundant.  If qubits are specified as the `Iterable['cirq.Qid']` type, that should exclude multi-dimensional ndarray-s.  I think we can request the highlighted_qubits to be a (1D) iterable of Qid-s and if users collect them to 2 or more dim arrays, they can easily flatten to a simple array themselves.

Please use `frozenset(qubits)` instead of `_as_set()`.